From b85a410e3404bb190889f6da99b50df228e3c1ef Mon Sep 17 00:00:00 2001
From: 5115 <nilimapradipm@maveric-systems.com>
Date: Fri, 19 Feb 2021 16:48:16 +0530
Subject: [PATCH] added patch file for demo

---
 src/main/java/HelloWorld.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main/java/HelloWorld.java b/src/main/java/HelloWorld.java
index 6637066..7039dd0 100644
--- a/src/main/java/HelloWorld.java
+++ b/src/main/java/HelloWorld.java
@@ -2,6 +2,6 @@ public class HelloWorld
 {
 public static void main(String[] args)
 {
-System.out.println("Hello World1");
+System.out.println("Hello World1 - patch demo");
 }
 }
\ No newline at end of file
-- 
2.30.0.windows.2

